frozen_mutation.hh

```diff
@@ -290,7 +290,7 @@ auto frozen_mutation::consume(schema_ptr s, frozen_mutation_consumer_adaptor<Con
     check_schema_version(schema_version(), *s);
     try {
         adaptor.on_new_partition(_pk);
-        partition().accept(s->get_column_mapping(), adaptor);
+        partition().accept_ordered(*s, adaptor);
```

Comment:
Why can't we leave the previous name? Do we want to keep an unordered consume variant too? For what? Won't that lead to mistakes?

Reply:
We have a few variants of `accept`, see https://github.com/scylladb/scylladb/blob/60e8f5743cc777882c6b53fa04a0e82c8ae862b2/mutation_partition_view.hh#L70-L75
The others may be used e.g. to unfreeze the mutation, and that doesn't need to be consumed in ordered since the mutation rebuilder reorders the range tombstones and rows when they are accepted into the mutation.
I think that doing that in order might hurt performance.

Reply:
We could rename accept() to accept_unordered() to reduce probability of mistakes.